test/framework/sandbox/sources/toy/toy-0.0_broken.patch

```diff
@@ -0,0 +1,10 @@
+--- a/toy-0.0/toy.source.orig   2013-11-30 14:43:48.000000000 +0100
```

Comment:
can you add a line on top describing why this patch is broken, and make clear it was done deliberately (to test apply_patch)